src/equations/compressible_euler_2d.jl

```diff
@@ -1991,6 +2012,28 @@ end
     return S
 end
 
+# Transformation from conservative variables u to d(s)/d(u)
+@inline function gradient_conservative(::typeof(entropy_math),
+                                       u, equations::CompressibleEulerEquations2D)
+    return cons2entropy(u, equations)
+end
+
+# Calculate specific entropy for conservative variable u
```

Comment:
```suggestion
# Calculate the specific entropy of Guermond et al. (2019): s_0 = p * rho^(-gamma) / (gamma-1).
# Note: This is *not* the "conventional" specific entropy s = ln(p / rho^(gamma)).
```
I think it wouldn't hurt to repeat this comment here

Reply:
Done in [27ded75](https://github.com/trixi-framework/Trixi.jl/pull/1792/commits/27ded75f469a66d6632815626d373549710bf1fc)